### ISSUE TITLE
Automatically to the next checklist

### ISSUE THREAD

**Issue by rohaimkhedr** [Ralphy444/Infinite-Checklist#9]
Maybe if you have done the checklist you get automatically to the next

**Comment by mrpidgeon** [comment]
yep this sounds good. maybe have a checkbox somewhere in a menu to turn it off though as it might annoy some people

**Comment by Ralphy444** [comment]
Hmmm, I'll se what I can do but it seems annoying tbh.
